File: patterns/C++/pattern4.cpp
/*
Generate the following pattern
1
2 2
3 3 3
4 4 4 4
5 5 5 5 5
*/

#include<bits/stdc++.h>
using namespace std;

void pattern3(int n){
    for(int i=1; i<=n; i++){
        for(int j=1; j<=i; j++){
            cout << i << " ";
        }
        cout << "\n";
    }
}

int main() {
    int n;
    cout << "Enter the number of rows(n): ";
    cin >> n;
    cout << "Here is the generated pattern\n";
    pattern3(n);

}

File: patterns/Python/pattern4.py
"""
Generate the following pattern
1
2 2
3 3 3
4 4 4 4
5 5 5 5 5
"""

def pattern4(n):
    for i in range(1,n+1):
        for j in range(i):
            print(i," ", end="")
        print("")
        
def main():
    i = int(input("Enter the number of rows: "))
    print("Here is the generated pattern\n")
    pattern4(i)
    
if __name__ == "__main__":
    main()